src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -295,6 +321,7 @@ void AddEnvironment ()
 					JniAddNativeMethodRegistrationAttributePresent = appConfState != null ? appConfState.JniAddNativeMethodRegistrationAttributePresent : false,
 					HaveRuntimeConfigBlob = haveRuntimeConfigBlob,
 					NumberOfAssembliesInApk = assemblyCount,
+					BundledAssemblyNameWidth = assemblyNameWidth + 1,
```

Comment:
Why the `+1` here?  Terminating NUL? If so, should `ApplicationConfigNativeAssemblyGenerator` do that addition instead?

Reply:
No, `ApplicationConfigNativeAssemblyGenerator` doesn't calculate the value - its task is just to write it.